packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.tsx

```diff
@@ -91,7 +91,7 @@ export class GroupHeader extends React.Component<IGroupDividerProps, IGroupHeade
 
           <div className='ms-GroupHeader-title ms-font-xl'>
             <span>{ group.name } </span>
-            <span>({ group.count }) </span>
+            <span>({ group.count }{ group.hasMoreData && '+' }) </span>
```

Comment:
i wonder if this is a localization concern. in RTL languages, should the plus go on the other side?